### DISCUSSION TITLE
detik.com - Banner notif breakingnews

### DISCUSSION THREAD

**Discussion by BanijaDev** [realodix/AdBlockID#1109] [Personal Filter]
https://inet.detik.com/security/d-4558794/aplikasi-download-video-ini-dituding-bahayakan-pengguna

![Screenshot_20220126-211208924.jpg](https://user-images.githubusercontent.com/97141433/151178568-07c25cfc-9d10-409a-af0d-32eefb348e3e.jpg)

**Comment by realodix** [comment]
Popup "adu perspektif" ya?

**Comment by 3xploiton3** [comment]
Sepertinya iya menutup konten

**Comment by realodix** [comment]
Sorry ini gak masuk ke cakupan AdBlockID

Anda bisa menambahkan ini

`detik.com##.notif_breakingnews`
